CHANGES/1033.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+Normalize default ports according to :rfc:`3986`.
+
+Specified ports are removed from the str representation of an URL if the port matches
+the schemes default port -- by :user:`commonism`.
```

Comment:
Let's link the mentioned structures and fix the grammar:
```suggestion
Specified ports are removed from the :class:`str` representation of a :class:`~yarl.URL`
if the port matches the scheme's default port -- by :user:`commonism`.
```

Reply:
@commonism I didn't commit this myself since I wanted you to preview it and check if I didn't confuse things. But I expect this to be the last bit to correct before merging.